x-pack/plugins/observability_solution/apm/server/routes/correlations/queries/fetch_duration_histogram_range_steps.ts

```diff
@@ -47,10 +51,14 @@ export const fetchDurationHistogramRangeSteps = async ({
   const steps = 100;
 
   if (durationMinOverride && durationMaxOverride) {
+    // these values should never be 0, so if they are we set them to 1
+    const durationMin = calculateDefaultDuration(durationMinOverride);
+    const durationMax = calculateDefaultDuration(durationMaxOverride);
```

Comment:
> [!TIP]
> I know it's still a draft.  You could also do the following if you want to avoid creating a new function:

```suggestion
    const durationMin = Math.min(1, durationMinOverride);
    const durationMax = Math.min(1, durationMaxOverride);
```

Reply:
Hey @crespocarlos thanks for the review!

Changed to use Math.min, good catch

Reply:
as @kpatticha pointed out, this should be `Math.max` instead.